### ISSUE TITLE
flaky: /fetch/content-type/response.window.html - fetch(): combined response Content-Type: text/plain */*

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno#12572]
https://github.com/denoland/deno/runs/4033473517?check_suite_focus=true#step:41:17211

Several failures on main because of it. 😢

**Comment by andreubotella** [comment]
We don't currently have any way to mark WPT tests as flaky other than to remove them from `expectations.json`. Adding a more flexible way to deal with flaky tests would also let #11460, which is currently blocked on flaky timeouts, move forward.

**Comment by lucacasonato** [comment]
I am going to disable the tests for now. I had not realized how flaky these specific tests are (50% failures or so...)

**Comment by ry** [comment]
I am very regularly getting these failures:
```
========================================

failures:

        "/dom/events/AddEventListenerOptions-passive.any.html - preventDefault should be ignored if-and-only-if the passive option is true"
        "/dom/events/AddEventListenerOptions-passive.any.html - passive behavior of one listener should be unaffected by the presence of other listeners"
        "/dom/events/AddEventListenerOptions-passive.any.worker.html - preventDefault should be ignored if-and-only-if the passive option is true"
        "/dom/events/AddEventListenerOptions-passive.any.worker.html - passive behavior of one listener should be unaffected by the presence of other listeners"
        "/dom/events/EventTarget-removeEventListener.any.html - removing a null event listener should succeed"
        "/dom/events/EventTarget-removeEventListener.any.worker.html - removing a null event listener should succeed"
        "/html/webappapis/structured-clone/structured-clone.any.html - A detached ArrayBuffer cannot be transferred"
        "/html/webappapis/structured-clone/structured-clone.any.html - A detached platform object cannot be transferred"
        "/html/webappapis/structured-clone/structured-clone.any.html - Transferring a non-transferable platform object fails"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - A detached ArrayBuffer cannot be transferred"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - A detached platform object cannot be transferred"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Transferring a non-transferable platform object fails"
        "/streams/idlharness.any.html - WritableStreamDefaultController interface: attribute abortReason"
        "/streams/idlharness.any.html - WritableStreamDefaultController interface: self.writableStreamDefaultController must inherit property \"abortReason\" with the proper type"
        "/streams/idlharness.any.worker.html - WritableStreamDefaultController interface: attribute abortReason"
        "/streams/idlharness.any.worker.html - WritableStreamDefaultController interface: self.writableStreamDefaultController must inherit property \"abortReason\" with the proper type"
        "/streams/writable-streams/aborting.any.html - WritableStreamDefaultController.signal"
        "/streams/writable-streams/aborting.any.worker.html - WritableStreamDefaultController.signal"

file failures:

        "/FileAPI/file/File-constructor.any.worker.html"

expected test failures that passed:

        "/html/webappapis/structured-clone/structured-clone.any.html - Array Blob object, Blob basic"
        "/html/webappapis/structured-clone/structured-clone.any.html - Array Blob object, Blob unpaired high surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.html - Array Blob object, Blob unpaired low surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.html - Array Blob object, Blob paired surrogates (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.html - Array Blob object, Blob empty"
        "/html/webappapis/structured-clone/structured-clone.any.html - Array Blob object, Blob NUL"
        "/html/webappapis/structured-clone/structured-clone.any.html - Array Blob object, two Blobs"
        "/html/webappapis/structured-clone/structured-clone.any.html - Object Blob object, Blob basic"
        "/html/webappapis/structured-clone/structured-clone.any.html - Object Blob object, Blob unpaired high surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.html - Object Blob object, Blob unpaired low surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.html - Object Blob object, Blob paired surrogates (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.html - Object Blob object, Blob empty"
        "/html/webappapis/structured-clone/structured-clone.any.html - Object Blob object, Blob NUL"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Array Blob object, Blob basic"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Array Blob object, Blob unpaired high surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Array Blob object, Blob unpaired low surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Array Blob object, Blob paired surrogates (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Array Blob object, Blob empty"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Array Blob object, Blob NUL"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Array Blob object, two Blobs"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Object Blob object, Blob basic"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Object Blob object, Blob unpaired high surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Object Blob object, Blob unpaired low surrogate (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Object Blob object, Blob paired surrogates (invalid utf-8)"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Object Blob object, Blob empty"
        "/html/webappapis/structured-clone/structured-clone.any.worker.html - Object Blob object, Blob NUL"
```
https://github.com/denoland/deno/runs/4219885165?check_suite_focus=true

**Comment by andreubotella** [comment]
> I am very regularly getting these failures:

Some of those failures are related to bugs that were fixed upstream (web-platform-tests/wpt#31150, web-platform-tests/wpt#31232), but those commits were pulled in #12682. It seems like @ry's branch in #12777 has an old checkout of wpt.

**Comment by lucacasonato** [comment]
These tests have been re-enabled.
